src/cgame/cg_main.cpp

```diff
@@ -1205,6 +1272,11 @@ void CG_Init( int serverMessageNum, int clientNum, const glconfig_t& gl, const G
 	// old servers
 	cgs.gameState = gameState;
 
+	/* Set level name and authors string to be used by the UI.
+	For some reason it must be done after cgs.gameState is set
```

Comment:
It's not a mystery - `cgs.gameState` is the array of config strings.

Reply:
Edited the comment.